androidPackageTut/android/app/src/main/java/com/androidpackagetut/MainActivity.java

```diff
@@ -1,30 +1,16 @@
 package com.androidpackagetut;
-import com.androidpackagetut.imagepicker.*; // import the package
 
 import com.facebook.react.ReactActivity;
-import com.facebook.react.ReactPackage;
-import com.facebook.react.shell.MainReactPackage;
-
-import java.util.Arrays;
-import java.util.List;
 
 public class MainActivity extends ReactActivity {
 
     @Override
     protected String getMainComponentName() {
-        return "androidpackagetut";
+        return "androidPackageTut";
```

Comment:
if you change the name of your component, you need to make changes in your android code aswell